core/modules/widgets/element.js

```diff
@@ -51,7 +51,12 @@ ElementWidget.prototype.render = function(parent,nextSibling) {
 	if(this.namespace) {
 		this.setVariable("namespace",this.namespace);
 	} else {
-		this.namespace = this.getVariable("namespace",{defaultValue: "http://www.w3.org/1999/xhtml"});
+		if (this.attributes.xmlns) {
+			this.namespace = this.attributes.xmlns;
+			this.setVariable("namespace",this.namespace);
+		} else {
+			this.namespace = this.getVariable("namespace",{defaultValue: tagNamespaces.body});
```

Comment:
Better to declare a variable `xhtmlNamespace` with the value `http://www.w3.org/1999/xhtml` and then use it here and in `tagNamespaces`.